src/wp-admin/includes/class-wp-site-health-auto-updates.php

```diff
@@ -74,9 +74,10 @@ public function test_constants( $constant, $value ) {
 		if ( defined( $constant ) && ! in_array( constant( $constant ), $acceptable_values, true ) ) {
 			return array(
 				'description' => sprintf(
-					/* translators: %s: Name of the constant used. */
-					__( 'The %s constant is defined and enabled.' ),
-					"<code>$constant</code>"
+					/* translators: 1: Name of the constant used. 2: Value of the constant used. */
+					__( 'The %1$s constant is defined as $2$s' ),
+					"<code>$constant</code>",
+					'<code>' . constant( $constant ) . '</code>'
```

Comment:
The value of the constant needs to be escaped, as it may contain unintended markup or similar.

```suggestion
					'<code>' . esc_html( constant( $constant ) ) . '</code>'
```